### PR TITLE
Timewindow

### PR DESCRIPTION
**Pull request by ianzhang366**
Addressing the time-window feature.

### CODE REVIEW

File: deploy/crds/app_v1alpha1_subscription_crd.yaml [hunk 1]
```diff
@@ -17,11 +17,12 @@ spec:
     listKind: SubscriptionList
     plural: subscriptions
     singular: subscription
-  scope: Namespaced
+  scope: ""
```
Comment (kuanf) [inline deploy/crds/app_v1alpha1_subscription_crd.yaml]: it should be namespaced!

Comment (ianzhang366) [reply deploy/crds/app_v1alpha1_subscription_crd.yaml]: it was changed by the `operator-sdk` command, is it that I have some wrong settings in my env, which leads to thise?

**Pull request merged**